marathon_acme/acme_util.py

```diff
@@ -31,3 +34,35 @@ def maybe_key(pem_path):
             )
         )
     return JWKRSA(key=key)
+
+
+@implementer(ICertificateStore)
+class MlbCertificateStore(DirectoryStore):
+    """
+    An ``ICertificateStore`` that wraps another ``ICertificateStore`` but
+    calls marathon-lb for a USR1 signal to be triggered when a certificate is
+    stored.
+    """
```

Comment:
Why does this need to be a `DirectoryStore` if it's a wrapper around another store?